tests/test_arr_venue_v2.py

```diff
@@ -4273,4 +4280,182 @@ def test_review_rating_forms(self, client, openreview_client, helpers, test_clie
+            message_button = status_table.find_element(By.CLASS_NAME, 'message-button')
+            message_button.click()
+            message_dropdown = message_button.find_element(By.CLASS_NAME, 'message-button-dropdown')
+            message_menu = message_dropdown.find_element(By.CLASS_NAME, 'dropdown-select__menu-list')
+
+            custom_funcs = message_menu.find_elements(By.XPATH, '*')
+
+            opts = [e for e in custom_funcs if e.text == email_option][0].click()
+            reviewer_msg_div = status_table.find_element(By.CLASS_NAME, 'ac-status-menu').find_element(By.ID, f'message-{role}s')
+            modal_content = reviewer_msg_div.find_element(By.CLASS_NAME, 'modal-dialog').find_element(By.CLASS_NAME, 'modal-content')
+            modal_body = modal_content.find_element(By.CLASS_NAME, 'modal-body')
+            modal_form = modal_body.find_element(By.CLASS_NAME, 'form-group')
+            email_body = modal_form.find_element(By.TAG_NAME, 'textarea')
+
+            modal_footer = modal_content.find_element(By.CLASS_NAME, 'modal-footer')
+            email_body.send_keys(email_option)  
+            next_buttons = modal_footer.find_element(By.CLASS_NAME, 'btn-primary')
+            next_buttons.click()
+            next_buttons.click()
+
+            time.sleep(0.5)
+
+        def users_with_message(email_option, members):
+            profile_ids = set()
+            email_map = { email : profile.id
+                for profile in openreview.tools.get_profiles(
+                    openreview_client,
+                    members
+                )
+                for email in profile.content['emails']
+            }
+            for email, id in email_map.items():
+                if any(message['content']['text'].startswith(email_option) for message in openreview_client.get_messages(to=email)):
+                    profile_ids.add(id)
+            return profile_ids
+
+        ac_email_options = [
+            'ACs with assigned checklists, none completed',
+            'ACs with assigned checklists, not all completed',
+        ]
+
+        area_chairs = openreview_client.get_group('aclweb.org/ACL/ARR/2023/August/Area_Chairs').members
+
+        ## Test 'ACs with assigned checklists, not all completed'
+        send_email('ACs with assigned checklists, not all completed', 'areachair')
+        emailed_users = users_with_message('ACs with assigned checklists, not all completed', area_chairs)
+
+        assignment_edges = {
+            group['id']['tail']: [edge['head'] for edge in group['values']] for group in openreview_client.get_grouped_edges(
+                invitation='aclweb.org/ACL/ARR/2023/August/Area_Chairs/-/Assignment',
+                groupby='tail',
+                select='head'
+            )
+        }
+
+        acs_with_missing_checklists = set()
+        # Check note data directly
+        for ac in area_chairs:
+            try:
+                assigned_ids = assignment_edges[ac]
```

Comment:
to avoid the KeyError you can use `.get` that returns None if the key doesn't exist.